modules/programs/zsh.nix

```diff
@@ -183,15 +183,22 @@ in
               type = types.str;
               description = ''
                 The name of the plugin.
-
-                Don't forget to add {option}`file`
-                if the script name does not follow convention.
               '';
             };
 
             file = mkOption {
               type = types.str;
-              description = "The plugin script to source.";
+              description = ''
+                The plugin script to source.
+                Required if the script name does not match {file}`name.plugin.zsh`
+                using the plugin {option}`name` from the plugin {option}`src`.
+              '';
+            };
+
+            completions = mkOption {
```

Comment:
should this just be a generic option instead of naming after completions?

Reply:
We can rename later, I suppose. Will just keep this as originally implemented, for now.